Felienne/hedy
# [CHORE] Fixed and improves some minor tutorial bugs

**Description**
In this PR we fix some minor issues and bugs that occurred during the user tutorial. We also make some small improvements to how we handle code snippets. The following most important issues are fixed:

- The default program is improved with underscores
- On explaining the output window the user code is parsed
- Better explain the run button step
- Set `window.State.unsaved_changes` to False when relevant
- Disabled next level button on explanation
- Disabled save / share button on top bar when explaining

**Fixes**
This PR fixes #2818.

**How to test**
Make sure you are logged in. Walk through the tutorial by visiting `/tutorial` and verify that all above summed up issues are indeed solved.
